kornia/losses/focal.py

```diff
@@ -78,11 +80,11 @@ def focal_loss(
         raise ValueError(f"input and target must be in the same device. Got: {input.device} and {target.device}")
```

Comment:
`KORNIA_CHECK(input.device == target.device, f"input and target must be in the same device. Got: {input.device} and {target.device}")`